Shared/analysis/LineOfSightController.h

```diff
@@ -18,16 +18,13 @@
 #define LINEOFSIGHTCONTROLLER_H
 
 // Qt headers
-#include <QUuid>
+#include <qtmetamacros.h>
```

Comment:
The use of QAbstractItemModel in the QPROPERTY below fails in the MOC step despite being forward declared and using Q_MOC_INCLUDE. Removing the QUuid since it was not longer necessary revealed this issue. I put this in to solve the issue but I think the better solution might just be to `#include <QAbstractItemModel>`.

Reply:
Found the proper combination for this include. QAbstractItemModel can be MOC included. Just had the wrong include....